x-pack/plugins/security_solution/public/cases/containers/configure/use_configure.test.tsx

```diff
@@ -165,6 +173,61 @@ describe('useConfigure', () => {
     });
   });
 
+  test('sort of happy save case configuration - getCaseConfigure', async () => {
```

Comment:
nit -> I think a better message would help a future developer to understand the purpose of the test